"public final Foo get_foo_with_err() throws Exception";
"public final Foo [] get_foo_arr_with_err() throws Exception";
"public final Foo [] get_foo_arr() {";
